src/content/docs/apm/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues.mdx

```diff
@@ -70,17 +87,37 @@ To enable Kafka event collection:
    ```
    SELECT average('producer-metrics.record-send-rate') from KafkaMetrics SINCE 30 minutes ago timeseries
    ```
+   If you are using Kafka Streams metrics, use the `KafkaStreamsMetrics` event type to access streams specific metrics.
+   
+## Enable Kafka Streams transactions [#collect-kafka-streams-transactions]
+If you are Kafka Streams, by default we do not enable transactions. This is to prevent unesessary overhead as Kafka applications tend to have high throughput. 
+
+Kafka Streams metrics unlike JMS metrics are not per record. Instead, a transaction begins when a Kafka consumer polls records and then the resulting data gets processed.
```

Comment:
```suggestion
Unlike JMS metrics, Kafka Streams metrics are not per record. Instead, a transaction begins when a Kafka consumer polls records and then the resulting data gets processed.
```

Reply:
Is the first sentence missing a word? Is it supposed to say "Kafka Streams metrics are not processed per record"?

Reply:
I think this should have referred to transactions, not metrics. Something like "Unlike KMS transactions, Kafka Streams transactions are not processed per record"